test/wrong/unknown-schema-format.yaml

```diff
@@ -0,0 +1,9 @@
+asyncapi: 2.1.0
+info:
+  version: "1.0.0"
+  title: "Test"
+channels:
+  mychannel:
+    publish:
+      message:
+        schemaFormat: "UnknownSchemaFormat"
```

Comment:
extend this test file with also broken message at the `component` level, and also add `subscribe` operation with message that uses `oneOf` and there are 2 messages with wrong format. We need to test all these cases